Makefile.in

```diff
@@ -1,7 +1,7 @@
-# Makefile.in generated by automake 1.14.1 from Makefile.am.
+# Makefile.in generated by automake 1.15.1 from Makefile.am.
```

Comment:
Should we remove the `autom4te.cache` directory and put it in `.gitignore` instead?

Reply:
Sounds safe to me after reading through: https://www.gnu.org/software/autoconf/manual/autoconf-2.66/html_node/Autom4te-Cache.html